lib/puppet/type/zone.rb

```diff
@@ -135,6 +151,10 @@ def up?
 	  sysconfig SCI tool on first boot of the zone."
   end
 
+  newparam(:zonecfg_archive) do
+    desc "Configure a zone from an Unified Archive"
+  end
+
```

Comment:
You can't leave this and archive below.  As above, why are you renaming the archive parameter. Your description is better but the rename doesn't make sense to me.